charter.html

```diff
@@ -294,16 +294,24 @@ <h2 id=work-items>Work Items</h2>
       <td><a href=https://github.com/annevk>Anne van Kesteren</a>
       <td>Various
     </tr>
-    <tr id=fps>
-      <th><a href=https://github.com/privacycg/first-party-sets>First-Party Sets</a>
-      <td><a href=https://github.com/krgovind>Kaustubha Govind</a>
-        &amp; <a href=https://github.com/HarneetSidhana>Harneet Sidhana</a>
-      <td><a href=https://www.w3.org/2011/webappsec/>WebAppSec</a>
+    <tr id=chips>
+      <th><a href=https://github.com/privacycg/CHIPS>Cookies Having Independent Partitioned State</a>
+      <td><a href=https://github.com/DCtheTall>Dylan Cutler</a> &amp;
+          <a href=https://github.com/krgovind>Kaustubha Govind</a>
+      <td>IETF
+    </tr>
+    <tr id=gpc>
+      <th><a href=https://github.com/privacycg/gpc>Global Privacy Control</a>
```

Comment:
URL: presuming we'll move the repo over & drop `-spec` as it seems unnecessary in this context.